lectures/inequality.md

```diff
@@ -523,8 +534,8 @@ T(p) = 1 - L (1-p)
     \approx \frac{\sum_{j\geq i} w_j}{ \sum_{j \leq n} w_j}, \quad i = [n (1-p)]
 $$(topshares)
 
-Here $[\cdot]$ is the greatest integer function, which rounds-off the real
-number inside the square bracket down to the integer less than the number.
+Here $\lfloor \cdot \rfloor$ is the floor function, which rounds-off the real
+number inside the square bracket down to the integer less than or equal to the number.
```

Comment:
"...which rounds any number down to the integer..."
"...the number" -> "that number"